### ISSUE TITLE
max-content  is not working in ie

### ISSUE THREAD

**Issue by Kelly-kml** [mdn/content#9312]

**Comment by vlwkaos** [comment]
Yes, MDN's [max-content](https://developer.mozilla.org/en-US/docs/Web/CSS/max-content) page lists IE as incomptible in the browser compatibility table. Could you give more detail on the problem you want resolved?

**Comment by AnilSeervi** [comment]
Internet Explorer doesn't support `max-content` property. - https://caniuse.com/mdn-css_properties_width_max-content
It doesn't support any of the Intrinsic & Extrinsic Sizing properties - https://caniuse.com/intrinsic-width
